charts/firefly/templates/_helpers.tpl

```diff
@@ -164,7 +177,11 @@ blockchain:
   type: ethereum
   ethereum:
     ethconnect:
+      {{ if .Values.ethconnect.enabled }}
```

Comment:
nice thanks for adding this, we'll need to add the same thing to the ERC1155 and ERC20 env vars as well.